qiskit/dagcircuit/dagnode.py

```diff
@@ -16,6 +16,17 @@
 
 import warnings
 
+from qiskit.circuit import Clbit
+
+
+def _condition_as_indices(operation, bit_indices):
+    cond = getattr(operation, "condition", None)
+    if cond is None:
+        return None
+    lhs, rhs = cond
```

Comment:
This is fine, but it might be easier to understand/follow if we say something like `bits, value` here. My first thought looking at this was lhs and rhs for an operator and only after looking at the usage below did I understand what was going on.

Reply:
Sounds good, done in 3dff489b1.